### PR TITLE
Vdb 1700 update geth in beta

### PR DESCRIPTION
**Pull request by elizabethengelman**
Update: here's the associated vdb-mcd-transformers PR: https://github.com/makerdao/vdb-mcd-transformers/pull/375/files

-------
This branch brings over changes that we made in production to:
- remove support for hashed keys since the prod geth patch does not send hashed keys - based on #120 
- adds support for the new geth patch in extractDiffs- based on #119
- removes the old geth patches from available options in extractDiffs' diff sources - based on #119 

I still need to make a corresponding change in vdb-mcd-transformers to make sure that extractDiffs is run from that repo instead, and that the filterQuery is created and passed to the geth subscription. This may be tricky, because as we now know the way we were referencing versioned contract names in the config file don't parse properly using the toml parser, but I think we can probably find a workaround short of needing to update how contract names are referenced in the config file.

I _think_ that other than the changes mentioned above in vdb-mcd-transformers that this may work, though I'd like to run it locally to verify. 
- since we're handling adding the hashed keys in the vdb keys_loader, and this PR removes it, there shouldn't be any changes necessary in vdb-mcd-transformers for hashed vs. unhashed keys. 
- the vdb-mcd-transformers in beta is still expecting hashed addresses, which we are taking care of here in `libraries/shared/storage/types/diff.go` - which hashes the addresses as it's formatting/decoding the diff it gets from the node.

### CODE REVIEW

File: cmd/extractDiffs.go [hunk 1]
```diff
@@ -29,34 +34,38 @@ func init() {
 	rootCmd.AddCommand(extractDiffsCmd)
 }
 
+func getContractAddresses() []string {
+	LogWithCommand.Info("Getting contract addresses from config file")
+	contracts := viper.GetStringMap("contract")
+	var addresses []string
+	for contractName := range contracts {
+		address := viper.GetStringMapString("contract." + contractName)["address"]
+		addresses = append(addresses, address)
```
Comment (rmulhol) [inline cmd/extractDiffs.go]: Wonder if it's worth tossing a `panic` in here if `address == ""`

**Review by rmulhol — DISMISSED** [review]
would like to take a longer look at this later but I'm fine with merging if it unblocks work in the transformers/actually building and testing the image

File: cmd/extractDiffs.go (hunk 1 above)
Comment (elizabethengelman) [reply cmd/extractDiffs.go]: Good call  - just added the CreateFilterQuery method that we have in prod as part of fe275dc

**Review by rmulhol — APPROVED** [review]
sorry meant to approve this earlier

**Pull request merged**